lib/20181211011437-create-article.js

```diff
@@ -0,0 +1,34 @@
+export default {
```

Comment:
Nice job @henperi but please what is this file doing in the lib folder? If I am correct this is a migration file